### DISCUSSION TITLE
How to connect search results with my geoserver.

### DISCUSSION THREAD

**Discussion by felipe-mendieta** [infra-geo-ouverte/igo2#815] [Q&A] [answered]
How do I connect my geoserver to igo2? I mean that when I search in the search bar "Search results" the results that appear are from my geoserver.

Do I have to build an api or can I do it some quick way?
Is there a ready-made api I can use?

**Comment by pelord** [comment, marked as answer]
The simpliest way for you is to build a WFS 2.0 StoredQuery and to use the StoredQueries Search Sources. 

There is no straitforward method at this moment to search into loaded vector data within the search bar. 

For most of IGO2 user, we have our own search service (iCherche, or other custom local search sources) based on data and indexed by elasticSearch. ....
